submission/submitter/src/main/kotlin/ac/uk/ebi/biostd/submission/submitter/request/SubmissionRequestLoader.kt

```diff
@@ -4,57 +4,40 @@ import ac.uk.ebi.biostd.persistence.common.model.RequestStatus.LOADED
 import ac.uk.ebi.biostd.persistence.common.model.SubmissionRequestFile
 import ac.uk.ebi.biostd.persistence.common.service.SubmissionRequestFilesPersistenceService
 import ac.uk.ebi.biostd.persistence.common.service.SubmissionRequestPersistenceService
-import ac.uk.ebi.biostd.persistence.filesystem.pagetab.PageTabService
 import ebi.ac.uk.extended.model.ExtFile
 import ebi.ac.uk.extended.model.ExtSubmission
 import ebi.ac.uk.extended.model.FireFile
 import ebi.ac.uk.extended.model.NfsFile
-import ebi.ac.uk.extended.model.allPageTabFiles
 import ebi.ac.uk.io.ext.md5
 import ebi.ac.uk.io.ext.size
 import mu.KotlinLogging
-import uk.ac.ebi.extended.serialization.service.FileProcessingService
 import java.time.OffsetDateTime
-import java.util.concurrent.atomic.AtomicInteger
 
 private val logger = KotlinLogging.logger {}
 
 class SubmissionRequestLoader(
     private val filesRequestService: SubmissionRequestFilesPersistenceService,
     private val requestService: SubmissionRequestPersistenceService,
-    private val fileProcessingService: FileProcessingService,
-    private val pageTabService: PageTabService,
 ) {
     /**
-     * - Calculate md5 and size for every file in submission request.
-     * - Generate submission pagetab
+     * Calculate md5 and size for every file in submission request.
      */
-    fun loadRequest(accNo: String, version: Int): ExtSubmission {
+    fun loadRequest(accNo: String, version: Int) {
```

Comment:
I still think consise solution is WAY better that this partitions. clearly the  complexity per line of code is higher as  it does  the same  things but in less code but is more mantianable, which in my opinion is our final goal.  

```
    fun loadRequest(accNo: String, version: Int) {
        val request = requestService.getIndexedRequest(accNo, version)
        val submission = request.submission
        filesRequestService
            .getSubmissionRequestFiles(submission.accNo, submission.version, request.currentIndex)
            .map { it.copy(file = loadFileAttributes(it.file)) }
            .forEach {
                filesRequestService.saveSubmissionRequestFile(it)
                requestService.updateRequestIndex(submission.accNo, submission.version, it.index)
            }
        requestService.saveSubmissionRequest(request.withNewStatus(status = LOADED))
    }
```